### PR TITLE
Fix format typo of missing a newline

### PR DESCRIPTION
**Pull request by 25077667**
The line 59 in devicemodel.c

### CODE REVIEW

File: examples/devicemodel.c [hunk 1]
```diff
@@ -56,7 +56,8 @@ static const struct dev_pm_ops devicemodel_pm_ops = {
     .poweroff = devicemodel_suspend,
     .freeze = devicemodel_suspend,
     .thaw = devicemodel_resume,
-    .restore = devicemodel_resume};
+    .restore = devicemodel_resume
```
Comment (jserv) [inline examples/devicemodel.c:59]: You have to specify the comma at the of this line. That is,
```c
.restore = devicemodel_resume,
```

Then, `clang-format` would indent as expected. Always run `clang-format -i` before submitting.

**Review by jserv — CHANGES_REQUESTED** [review]
Rebase the latest master branch.

**Comment by 25077667** [comment]
I meet a problem that the lkmpg.tex could not be applied as 65a8c12 version.
It still have a changes from Jason Huang, f6d4180.

**Comment by jserv** [comment]
> I meet a problem that the lkmpg.tex could not be applied as [65a8c12](https://github.com/sysprog21/lkmpg/commit/65a8c123e4002b9143dc07c1784a2a12b1130c4f) version.
> It still has changes from Jason Huang, [f6d4180](https://github.com/sysprog21/lkmpg/commit/f6d4180ed9f98d0f11ad03b0b280120748f71f19).

You can do the following:
```shell
git format-patch -1
git remote add upstream https://github.com/sysprog21/lkmpg
git fetch upstream
git checkout -b style-tweaks upstream/master
git am *.patch
git push origin style-tweaks
```

 Then, edit this pull request to specify the appropriate branch.

**Pull request closed without merging**